SpatialGDK/Documentation/unreal-features-support.md

```diff
@@ -319,8 +319,8 @@ Support of Unreal features with the GDK in a single server-worker configuration:
  <tr>
     <td rowspan="3"><a href="https://docs.unrealengine.com/en-us/Gameplay/Networking/Travelling"><b>Travel<b></a></td>
     <td>Server Travel</td>
-    <td class="unplanned"></td>
-    <td>Server Travel is on the <a href="https://github.com/spatialos/UnrealGDK/projects/1#card-22461878"> roadmap</a> but is not currently supported.</td>
+    <td class="supported"></td>
+    <td>Not supported when used in conjunction with zoning or offloading</td>
```

Comment:
Personally, I would have all of the travels as `ServerTravel` and `ClientTravel` and `SeamlessTravel` to be consistent, but I don't want to change the whole doc here.

Suggest use consistent terms and info.

```suggestion
    <td>Not supported with multiserver zoning or offloading. Multiserver support is on the [roadmap](https://github.com/spatialos/UnrealGDK/projects/1#card-22461878).</td>
```